tools/awslint/lib/rules/cfn-resource.ts

```diff
@@ -67,13 +66,6 @@ export class CfnResourceReflection {
 
     this.namespace = fullname.split('::').slice(0, 2).join('::');
 
-    // special-case
-    const basename = this.basename
```

Comment:
We still need these special cases for rendering the L2 linter

Reply:
yes - I thought they would come back, but I was in a clean everything out.